src/components/ChatApp/words.json

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
To be honest, this don't really give a personlaized recomendation based on user typing patterns, thus making the whole feature reduntant. This feature will need more discussion to be even considered "required".

Reply:
Yes, you are right. It does not give results based on user typing pattern. It is a very basic feature in which the last word of the input stream is compared with an array of 1000 commonly used words so that users does not need to type entire words.